README.md

```diff
@@ -66,7 +66,8 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-The link will be added soon.
+- [Live Demo](pocketpal-vqre.onrender.com)
+- [Live Video](https://www.loom.com/share/db41602983204fdc98004470d4696c38)
```

Comment:
- You have a good application , and a good presentation 😉  but please remember to highlight  piece of the code the you find interesting or  proud of making.

![image](https://user-images.githubusercontent.com/75126481/236606234-a1065b1f-7514-481a-867f-aa4655842882.png)